packages/pact-testing/src/helpers/shell-helpers.sh

```
@@ -0,0 +1,225 @@
+
+    local GO_VERSION=$(go version)
+    log_success "Go found: $GO_VERSION"
+}
+
+# Build BFF server
+build_bff_server() {
+    local BFF_DIR="$1"
+    local TEST_RUN_DIR="$2"
+    
+    log_info "Building Mock BFF server..."
+    cd "$BFF_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)